frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -62,6 +66,22 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
     resource?.spec?.storageClassName,
   );
 
+  const allowedAccessModes = () => {
+    if (
+      scResource?.provisioner.includes(cephRBDProvisionerSuffix) &&
+      resource.spec.volumeMode !== 'Block'
+    ) {
+      return ['ReadWriteOnce'];
+    }
+    return (
+      provisionerAccessModeMapping[scResource?.provisioner] ??
```

Comment:
Why not just update the `provisionerAccessModeMapping` with ceph provisioners & now have `getAccessModeForProvisioner` itself.

Also let's update it for `ebs.csi.aws.com` CSI based gp2 class as well.